App/src/components/NewCalendar.js

```diff
@@ -251,9 +260,22 @@ function NewCalendar({
                         {onShiftClick ? (
                           <button
                             css={{
-                              background: "none",
-                              border: "none",
-                              cursor: "pointer"
+                              ...buttonCss,
+                              background:
```

Comment:
This background validation is not required anymore since you are using `selectedUsers` and `selectedDate`, ensure you pass an array of numbers for `selectedUsers` and a valid date string as `selectedDate` (you already did this one) and it will already highlight the selected dates.

Reply:
If you remove the background validation move the content of `buttonCss` back here.